soc/intel/intel_adsp/ace/Kconfig.soc

```diff
@@ -20,7 +20,7 @@ config SOC_INTEL_ACE20_LNL
 	help
 	  ACE 2.0 Lunar Lake PCH
 
-config SOC_INTEL_ACE30_PTL
+config SOC_ACE30
```

Comment:
Please keep the Intel name here to act as namespace: `SOC_INTEL_ACE30`.

Reply:
Changed to ```SOC_INTEL_ACE30```